coremltools/converters/mil/frontend/torch/torch_op_registry.py

```diff
@@ -127,3 +129,18 @@ def func_wrapper(func):
         # decorator called without argument
         return func_wrapper
     return func_wrapper(_func)
+
+
+def is_torch_fx_node_supported(
```

Comment:
Shall we also surface this symbol in the higher up modules/packages to make the import command for this API simpler?

Reply:
Yeah I would love to surface it to `ct.is_torch_fx_node_supported`

Reply:
I think we should think twice about this:
If you look at the exposed API for `ct.xxx`:
https://github.com/apple/coremltools/blob/main/coremltools/test/api/test_api_visibilities.py#L21
All of them have the traits of:
1. coreml model level
2. shared across all frontends
As the result, by inserting this torch specific utils in the high level, is not consistent with the design logic, at least by far.